LayoutTests/platform/ios/TestExpectations

```diff
@@ -5190,6 +5190,7 @@ editing/selection/click-in-margins-inside-editable-div.html [ Failure ]
 editing/selection/click-in-padding-with-multiple-line-boxes.html [ Failure ]
 editing/selection/double-click-selection-with-dom-mutation.html [ Failure ]
 editing/selection/extend-by-character-006.html [ Failure ]
+editing/selection/extend-by-character-007.html [ Failure ]
```

Comment:
Can you also update the test expectation file for this on iOS rather than marking it as failure?